apps/frontend/src/components/Map/Map.tsx

```diff
@@ -0,0 +1,88 @@
+import { AspectRatio, Stack } from '@chakra-ui/layout';
+import * as React from 'react';
+import { dRaw, useData, zap } from '../../context/DataContext';
+import { Waypoint } from '../../types';
+import { powerColor } from '../../colors';
+import { Box } from '@chakra-ui/react';
+
+interface Props {}
+
+const waypointsToSvgPoints = (waypoints: Waypoint[], multiplier = 1) =>
```

Comment:
pretty nitpicky bikeshedding, but could this be at the bottom, if allowed by ts?

Reply:
Agreed!